installing/installing_azure/installing-azure-account.adoc

```diff
@@ -44,3 +46,5 @@ xref:../../installing/installing_azure/installing-azure-customizations.adoc#inst
 or
 xref:../../installing/installing_azure/installing-azure-default.adoc#installing-azure-default[quickly install a cluster]
 with default options.
+
+* link:https://learn.microsoft.com/en-us/azure/role-based-access-control/custom-roles[Azure custom roles]
```

Comment:
We have to add a link here since we have a note that suggests user to check azure docs. https://github.com/openshift/openshift-docs/pull/56274/files#r1121582526

Reply:
Addressed

Reply:
Please ignore my previous comment - we can add external links in the modules